bootstrap.sh

```diff
@@ -13,47 +13,41 @@ rm -f swig/vipsCC/*.cxx
 rm -f swig/vipsCC/VImage.h
 rm -f swig/vipsCC/VImage.py python/vipsCC/VError.py python/vipsCC/VMask.py python/vipsCC/Display.py
 rm -f benchmark/temp*
-( mkdir poop ; \
-	mv doc/libvips-docs.xml.in poop ; \
-	mv doc/Makefile.am poop ; \
-	mv doc/images poop ; \
-	mv doc/*.xml poop ; \
-	mv doc/*.py poop ; \
-	rm -rf doc/* ; \
-	mv poop/* doc ; \
-	rmdir poop \
-)
+find doc \( \
+  -not -path doc/libvips-docs.xml \
+  -and -path doc/libvips-docs.xml.in \
+   -or -path doc/Makefile.am \
+   -or -path 'doc/images/*' \
+   -or -depth 1 -name '*.xml' \
+   -or -depth 1 -name '*.py' \
+\) -prune -or -delete
 
-# glib-gettextize asks us to copy these files to m4 if they aren't there
-# I don't have $ACDIR/isc-posix.m4, how mysterious
 ACDIR=`aclocal --print-ac-dir`
-
-# OS X with brew sets ACDIR to
-# /usr/local/Cellar/automake/1.13.1/share/aclocal, the staging area, which is
-# totally wrong argh
-if [ ! -d $ACDIR ]; then
-	ACDIR=/usr/local/share/aclocal
+# OS X with brew has a dirlist in ACDIR that points to several directories
+# dirlist supports wildcards, but that would require eval ... which is evil
+if [ -e $ACDIR/dirlist ]; then
+  ACDIR=`cat $ACDIR/dirlist`
 fi
 
 mkdir -p m4
-cp $ACDIR/codeset.m4 m4
-cp $ACDIR/gettext.m4 m4
-cp $ACDIR/glibc21.m4 m4
-cp $ACDIR/iconv.m4 m4
-cp $ACDIR/lcmessage.m4 m4
-cp $ACDIR/progtest.m4 m4
-cp $ACDIR/introspection.m4 m4
+# glib-gettextize asks us to copy these files to m4 if they aren't there:
+files="codeset gettext glibc21 iconv isc-posix lcmessage progtest introspection"
```

Comment:
I have none of these files except introspection.m4 in /usr/local/share/aclocal and doc generation seems to work fine.
Is this actually required on linux?